pulsar-broker/src/test/java/org/apache/pulsar/client/impl/NegativeAcksTest.java

```diff
@@ -106,6 +106,7 @@ public void testNegativeAcks(boolean batching, boolean usePartitions, Subscripti
         log.info("Test negative acks batching={} partitions={} subType={} negAckDelayMs={}", batching, usePartitions,
                 subscriptionType, negAcksDelayMillis);
         String topic = BrokerTestUtil.newUniqueName("testNegativeAcks");
+        admin.topics().createPartitionedTopic(topic, 2);
```

Comment:
Should apply `usePartitions` to run test on partitioned topic and non-partitioned topic